src/main/java/org/example/repository/BuildingRepository.java

```diff
@@ -0,0 +1,9 @@
+package org.example.repository;
+
+import org.example.entity.Building;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface BuildingRepository extends JpaRepository<Building, Integer> {
+
+    Building findBuildingById(int id);
```

Comment:
You cour remove this method from repository and use standart `findById` from JpaRepository